cypress-io/cypress
# fix: escape backslashes in downloadFolder for Firefox on Windows #17896

- Closes #17896

### User facing changelog
n/a

### Steps to test
- Clear Firefox Cypress profile (important!.. a correct folder path may have been set there..)
- In https://github.com/mirobo/cypress-fiddle/blob/main/cypress/plugins/index.js remove the workaround for downloadDir
- Start the test firefox-download.spec.js that downloads a file (doesnt assert anything).. you'll see the wrong folder. Activate workaround and it works. Tested it briefly by changing the JS-file in the Cypress app dir directly.


### How has the user experience changed?
no

### PR Tasks
- [ ] Have tests been added/updated? _No, there aren't any tests at all.._
